### PR TITLE
Move the blog to something more Kinto related.

### PR DESCRIPTION
**Pull request by Natim**
r? @leplatrem @glasserc @magopian

### CODE REVIEW

**Comment by Natim** [comment]
I don't have the permission anymore to move the blog to the Kinto org. If you agree with this proposal, please do so after merging this PR.

**Comment by leplatrem** [comment]
So your proposal is to move the repo to the Kinto org? Should the blog move to kinto-storage.org too?

File: content/pages/about.rst [hunk 1]
```diff
@@ -5,15 +5,14 @@
 :lang: fr
 :url: a-propos
 
-Nous sommes une équipe «Cloud-services» partiellement francophone à
-`Mozilla <https://www.mozilla.com/>`_ :
-`Benson Wong <https://twitter.com/mostlygeek>`_,
-`Ben Hearsum <https://twitter.com/bhearsum>`_,
+Nous sommes un groupe de développeurs partiellements francophones qui gravitons autour de Kinto :
```
Comment (leplatrem) [inline content/pages/about.rst]: partiellement

File: pelicanconf.py [hunk 2]
```diff
@@ -21,9 +21,6 @@
 
 DEFAULT_LANG = u'fr'
 
-# Social widget
-SOCIAL = (('Github', 'https://github.com/mozilla-services'),)
```
Comment (leplatrem) [inline pelicanconf.py:25]: github.com/Kinto ?

Comment (Natim) [reply pelicanconf.py:25]: Yes but this is redefined below

**Comment by Natim** [comment]
>  Should the blog move to kinto-storage.org too?

I don't think so at least for now, servicedenuages.fr is already not linked to mozilla-services

**Pull request closed without merging**